fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -250,6 +251,7 @@ def enumClasses(PogoDeviceClass cls) '''
 		"""Python enumerated type for «attr.name.toFirstUpper» attribute."""
 		«ENDIF»
 	«ENDIF»
+	«ENDIF»

```

Comment:
The inherited class has too many blank lines after the attributes section header:
```python
    # ----------
    # Attributes
    # ----------




    testMode = attribute(
        dtype=TestMode,
    )
```

Just a single line, like the base device, would be better:
```python
    # ----------
    # Attributes
    # ----------

    healthState = attribute(
        dtype=HealthState,
    )
```

Reply:
Yes agree. Will update that.